satsense/features/pantex.py

```diff
@@ -75,6 +46,19 @@ def get_rii_dist_angles():
     return get_rii_dist_angles.offsets
 
 
+def __get_rii_dist_angle(pixels_dist):
```

Comment:
Use one leading underscore only for non-public methods and instance variables. [PEP8](https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables)
```suggestion
def _get_rii_dist_angle(pixels_dist):
```